Dockerfile.web

```diff
@@ -9,7 +9,7 @@ FROM registry.access.redhat.com/ubi8/nodejs-16 as static
 USER 0
 
 ADD web .
-RUN npm i yarn && yarn install --frozen-lockfile
+RUN npm i yarn && NODE_OPTIONS=--max_old_space_size=1000 yarn install --frozen-lockfile
```

Comment:
Can we round this to 1024 bytes? Also some sizing suggestions: https://stackoverflow.com/a/63495296